### PR TITLE
Patient model and test added

### PR DESCRIPTION
**Pull request by kristianmatzen**
Patient model added with id, token, name and email

Model and test is based on carer model and test

### CODE REVIEW

**Comment by loklaan** [comment]
Could you add more simple tests for the model? Like separate tests for.. updating.. destroying, etc. It would just look good on the coverage, but wouldn't be hard to write.

**Comment by kristianmatzen** [comment]
Done... Also put in the coordinates for the patient model...

File: karma.conf.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ module.exports = function(config) {
     ],
     files: [
       // bower#dev:js
+      'public/libs/jquery/dist/jquery.js',
```
Comment (loklaan) [inline karma.conf.js]: Hahaha I was wondering who would be the first to commit these injected dependencies.

File: tests/server/models-spec.js [hunk 2]
```diff
@@ -31,9 +31,102 @@ describe('DemS models', function() {
...
+      var newPatient = {
+        id : 11
+      };
+
+      patientModel.create(newPatient, function(err, patient) {
+        expect(err).toBe(null);
+        expect(patient.id).toBe(11);
+      });
+      
+      patientModel.count(function(err, count) {
```
Comment (loklaan) [inline tests/server/models-spec.js]: All the other calls in this test should be in this guys callback.

**Pull request merged**